### PR TITLE
Download the stage0 dotnet tools from the beta feed.

### PR DESCRIPTION
**Pull request by Sridhar-MS**
Set the correct channel so that the stage0 binaries are downloaded correctly.

    ** This commit cannot be directly merged from the release branch to the master branch **

/cc: @brthor

### CODE REVIEW

File: scripts/compile/compile-stage.sh [hunk 1]
```diff
@@ -118,4 +118,4 @@ fi
 # Stamp the output with the commit metadata
 COMMIT=$(git rev-parse HEAD)
 echo $COMMIT > $OUTPUT_DIR/.version
-echo $DOTNET_BUILD_VERSION >> $OUTPUT_DIR/.version
+echo $DOTNET_CLI_VERSION >> $OUTPUT_DIR/.version
```
Comment (brthor) [inline scripts/compile/compile-stage.sh]: good catch :+1:

**Comment by brthor** [comment]
LGTM

**Comment by Sridhar-MS** [comment]
@dotnet-bot test OSX Debug Build please

**Comment by Sridhar-MS** [comment]
@dotnet-bot test OSX Release Build please

**Pull request merged**